_posts/2017-01-05-enhancing-your-webcam.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title:  "Enhancing your webcam using canvas.captureStream()"
+date:   2017-01-05 12:00:00
+author: "Zombocam"
```

Comment:
Afaict the convention on this blog is to use the name of the author here (as opposed to the "marketing entity").